tensorflow_encrypted/config.py

```diff
@@ -245,13 +247,15 @@ def session(
         master: Optional[Union[int, str]] = None,
         log_device_placement: bool = False
     ) -> tf.Session:
-        # cpu_cores = get_cpu_quota_within_docker() or multiprocessing.cpu_count()
+        cpu_cores = get_cpu_quota()
```

Comment:
should we only specify this if we're inside docker or is it generally useful (got the impression that TF is already making some choices here)?

Reply:
Now that you are mentioning it, we should probably start by avoiding setting it outside docker until we have a strong understanding of possible related optimizations.

I'll update this part so it returns `None` outside of docker.